be/src/runtime/exec_env_init.cpp

```diff
@@ -208,6 +215,10 @@ void ExecEnv::_init_buffer_pool(int64_t min_page_size,
 }
 
 void ExecEnv::_destory() {
+    //Only destroy once after init
+    if (!_is_init) {
```

Comment:
if ExecEnv::_init return failed status, then you will be memory leak.  for examaple 124 line in this file  return error.

Reply:
If is_init is not judged, the initial initialization fails, and the destroy will be CoreDump later